conduit/src/Data/Conduit/Combinators.hs

```diff
@@ -304,6 +301,21 @@ import           Data.Sequences (LazySequence (..))
 --   so there's no way for the conduit to guarantee a certain amount
 --   of demand from the upstream.
 
+-- $naming_scheme
+--
+-- [@Functions ending with C@]: This postfix indicates a function returns a
```

Comment:
Note that this applies to the `Conduit` module, but _not_ `Data.Conduit.Combinators`. That should probably be clarified here.